rpm/tarantool-memcached.spec

```diff
@@ -1,5 +1,5 @@
 Name: tarantool-memcached
-Version: 0.1.0
+Version: 1.0.1
```

Comment:
This field is automatically set by packpack, however I don't mind to keep it actual for ones, who want to build a package in another way. There is a little pitfall, though. `master` may be essentially updated between releases and so will not be equivalent to 1.0.1. I think, it should be `1.0.2-dev` or kinda.

I would exclude the change from the PR and let @LeonidVas to update versions and changelogs in packages, if he is okay with the extra effort for releasing the module.